.yetus-excludes

```diff
@@ -0,0 +1 @@
+.*\.txt
```

Comment:
We don't want that. 
Why?

Reply:
Because `yetus` make a lot of crazy checking about tabs in escripts. But we need tabs in this or may be another test, just because we use `mysql` output with tabs to check the test results.

Reply:
the we should subset it to just escripts folders -- disabling it everywhere seems too agressive

Reply:
OK -- `yetus` check is now disabled only for MariaDB escript.

Reply:
Fine